android/src/main/java/com/pdftron/pdftronflutter/helpers/PluginUtils.java

```diff
@@ -1485,12 +1506,15 @@ private static void setToolMode(String toolModeString, MethodChannel.Result resu
         result.success(null);
     }
 
-    private static void setLeadingNavButtonIcon(String leadingNavButtonIcon, MethodChannel.Result result, ViewerComponent component) {
```

Comment:
What happened to `setLeadingNavButtonIcon` ?

Reply:
That is moved to FlutterDocumentView and FlutterPlugin respectively. I decided not to have that as a shared function in a previous PR, but forgot to remove this one which was not used anymore.